traffic_ops/install/bin/_postinstall

```diff
@@ -218,7 +234,9 @@ sub generateCdnConf {
             $#secrets = $cdnConfiguration{keepSecrets} - 1;
         }
     }
+    $cdnConf = setCdnConfGoPort($cdnConf);
```

Comment:
Need another subroutine to account for setting the `max_db_connections` in postinstall (or figuring out reasonable defaults) as discussed